vulcanai2/tests/plotters/test_visualization.py

```diff
@@ -104,3 +113,47 @@ def test_compute_saliency_map_dnn(self, dnn_class):
 
         # Test hook removal
         assert dnn_class._backward_hooks == model_copy._backward_hooks
+
+    def test_display_saliency_overlay(self, dnn_class):
+        curr_path = str(os.path.dirname(__file__)) + '/'
+        img = np.zeros((25,25))
+        test_input_1B = torch.ones([1, *dnn_class.in_dim])
+        sal_map_1B = np.array(compute_saliency_map(
+            dnn_class,
+            test_input_1B, torch.LongTensor([2])))
+        display_saliency_overlay(img, sal_map_1B, shape=(25,25), save_path=curr_path)
+        for file in os.listdir(curr_path):
```

Comment:
Unless I'm mistaken, there's no way to fail this loop?

Reply:
has this been addressed @joemehltretter ?

Reply:
@rfratila Yes, the addition of the `file_created` variable fixed this issue.

Reply:
ok great!